ggml-cuda/dmmv.cu

```diff
@@ -563,12 +574,15 @@ static void dequantize_mul_mat_vec_q3_K_cuda(const void * vx, const float * y, f
     dequantize_mul_mat_vec_q3_k<<<block_nums, block_dims, 0, stream>>>(vx, y, dst, ncols, nrows);
 }
 
+#define BLOCK_DIM_X 32
+#define BLOCK_DIM_Y 4
```

Comment:
```suggestion
#define Q4_K_BLOCK_DIM_X 32
#define Q4_K_BLOCK_DIM_Y 4
```

Reply:
Also if the optimal value for the x dimension is 32, should that parameter simply be removed?

Reply:
Any is fine with me - had them there as defines to easily change them and also reuse them across other kernels in my testing. 
I left them in this PR for readability but happy to hard-code them or renaming them.

Reply:
In that case I would just hard-code them because as it is they get multiplied with magic numbers anyways.